### PR TITLE
Add new batch of items in palette for feast branding

### PR DESCRIPTION
**Pull request by Divs-B**
## What's changed?
Items request to add in existing palatte to support editorial team for feast branding.

ref of task:https://trello.com/c/6p3Ps72z/3436-feast-branding-new-items-to-add-in-palette

image

## Implementation notes
- Upload images to S3 via https://s3-uploader.gutools.co.uk/upload
- Add items in their alphabetical order with these image path in `feastPalette`

### General
- [ ] 🤖 Relevant tests added
- [X] ✅ CI checks / tests run locally
- [ ] 🔍 Checked on CODE

### Client
- [X] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [X] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [X] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

File: fronts-client/src/constants/feastPalettes.ts [hunk 1]
```diff
@@ -801,6 +982,36 @@ export const feastCollectionPalettes: PaletteOption[] = [
...
+			dark: {
+				foregroundHex: '#F0CDB5',
+				backgroundHex: '#363632',
+			},
+		},
+		imageURL: 'https://uploads.guim.co.uk/2024/11/18/Utensils_1.png',
+	},
+	{
+		id: 'utensils_2',
+		name: 'Utensils 2',
```
Comment (Divs-B) [inline fronts-client/src/constants/feastPalettes.ts:1002]: I think `Utensil 1` and `Utensil 2` are okay in terms of display names and distinguishing between two.
or let me know if we need just `Utensil`  for both?

Comment (fredex42) [reply fronts-client/src/constants/feastPalettes.ts:1002]: I agree 😁

**Review by fredex42 — APPROVED** [review]
🎨 🧑‍🎨

File: fronts-client/src/constants/feastPalettes.ts [hunk 2]
```diff
@@ -141,6 +141,52 @@ export const feastCollectionPalettes: PaletteOption[] = [
...
+			dark: {
+				foregroundHex: '#FCF0F6',
+				backgroundHex: '#363632',
+			},
+		},
+		imageURL: ' https://uploads.guim.co.uk/2024/11/18/Berries.png',
+	},
+	{
+		id: 'blackeberries_blueberries',
+		name: 'Blackeberries\nBlueberries',
```
Comment (emdash-ie) [inline fronts-client/src/constants/feastPalettes.ts:161]: There’s a spelling mistake here: the correct spelling would be `blackberries_blueberries`, without the extra e. Can we change it, or does it need to be this way to match an existing name somewhere else?

Comment (Divs-B) [reply fronts-client/src/constants/feastPalettes.ts:161]: Ah nice spot! Yes, easy to change. I will remove extra `e` from `Blackeberries` 😀

**Comment by Divs-B** [comment]
Looks fine on CODE too.

File: fronts-client/src/constants/feastPalettes.ts (hunk 2 above)
Comment (Divs-B) [reply fronts-client/src/constants/feastPalettes.ts]: Thanks @emdash-ie, plz recheck when you can

**Review by emdash-ie — APPROVED** [review]

**Pull request merged**